cg/meta/backup/backup.py

```diff
@@ -404,25 +404,32 @@ def mark_file_as_archived(self, spring_file_path: Path) -> None:
             LOG.info(f"Dry run, no changes made to {spring_file_path}")
             return
         hk_spring_file: File = self.hk_api.files(path=str(spring_file_path)).first()
-        LOG.info(f"Setting {spring_file_path} to archived in Housekeeper")
-        self.hk_api.set_to_archive(file=hk_spring_file, value=True)
-
-    def is_to_be_retrieved_and_decrypted(self, spring_file_path: Path) -> bool:
-        """Determines if a spring file is archived on PDC and needs to be retrieved and decrypted."""
-        return (
-            self.hk_api.files(path=str(spring_file_path)).first().to_archive
-            and not spring_file_path.exists()
-        )
+        if hk_spring_file:
+            LOG.info(f"Setting {spring_file_path} to archived in Housekeeper")
+            self.hk_api.set_to_archive(file=hk_spring_file, value=True)
+        else:
+            LOG.warning(f"Could not find {spring_file_path} in Housekeeper")
 
     def remove_archived_spring_file(self, spring_file_path: Path) -> None:
         """Removes all files related to spring PDC archiving."""
         if not self.dry_run:
             LOG.info(f"Removing spring file {spring_file_path} from disk")
             spring_file_path.unlink()
 
+    def is_to_be_retrieved_and_decrypted(self, spring_file_path: Path) -> bool:
+        """Determines if a spring file is archived on PDC and needs to be retrieved and decrypted."""
+        file = self.hk_api.files(path=str(spring_file_path)).first()
+        if file and not spring_file_path.exists():
+            LOG.warning(f"Could not find {spring_file_path} in Housekeeper")
```

Comment:
I think we could be more specific here - since the problem here is not that we cant find the file in the database but the fact that it does not exist on disk any longer